src/main/java/com/wcc/platform/domain/cms/attributes/Experience.java

```diff
@@ -16,6 +16,16 @@ public enum Experience {
 
   @NotBlank private final String experienceRange;
 
+  public static Experience fromYears(Integer years) {
+    if (years == null || years <= 0) return NO_EXPERIENCE;
+    return switch (years) {
+      case 1, 2, 3, 4, 5 -> YEARS_1_TO_5;
```

Comment:
I think we discussed with frontend team we don't need to add the enum, I don't think if you were in the meeting the same day.